task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawerComponents/TaskFormDatePickers.vue

```diff
@@ -78,6 +78,7 @@
                 value="left"
                 class="my-0"
                 small
+                :disabled="testToday"                
```

Comment:
Please change the function name to use something which explain what the function do.
It can be isActive, or canChooseToday, or something else, but not containing test